dockyard/common/link.py

```diff
@@ -0,0 +1,20 @@
+
+def make_url(host='127.0.0.1', port=None, protocol='http', url=None):
+    head = protocol + '://'
```

Comment:
I want something better for these two statement. but i don't know, what can be the better statements may be something like this :100: 

("%s://%s)%(--,--)